onnx/defs/nn/defs.cc

```diff
@@ -1542,4 +1534,74 @@ ONNX_OPERATOR_SET_SCHEMA(
         })
         .SetDoc(TfIdfVectorizer_ver9_doc));
 
+static const char* StringNormalizer_ver10_doc = R"DOC(
+StringNormalization performs string operations for basic cleaning. 
+This operator has only one input (denoted by X) and only one output 
+(denoted by Y). This operator first examines the elements in the X, 
+and removes elements specified in "stopwords" attribute. 
+After removing stop words, the intermediate result can be further lowercased, 
+uppercased, or just returned depending the "case_change_action" attribute.
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    StringNormalizer,
+    10,
+    OpSchema()
+        .Input(0, "X", "UTF-8 strings to normalize", "tensor(string)")
```

Comment:
Shall we explicitly say that input's shape should [C] and [1, C]? Also, I am wondering why do we need to support [1, C] shape. Sounds a bit redundant to me. Since we always add an unsqueeze before.

Reply:
In traditional ML pipelines in, for example, scikit-learn and Core ML, batch size 1 is very common. It's ok to remove but I'd like to support it to avoid adding extra ops. In addition, the reason that we can't have batch size > 1 is the lack of sparse tensor. This restriction will be eventually removed, I believe.

Reply:
Okay, let's keep it as it is now. But please explicitly describe what input shape will be accepted.

Reply:
So the doc string clearly states that but I will also mention this in the input spec.